lxd/instance/drivers/driver_qemu.go

```diff
@@ -572,6 +572,25 @@ func (d *qemu) onStop(target string) error {
 	// Reset timeout to 30s.
 	op.Reset()
 
+	// Wait for QEMU process to end (to avoiding racing start when restarting).
+	d.logger.Debug("Waiting for VM process to finish")
+	waitDuration := time.Duration(time.Second * time.Duration(5))
```

Comment:
I think we could probably bump that to 20s to give us more time to flush any pending data to disk.
We'll probably want to call op.Reset() after it though.

Reply:
Anyway, something that can be tweaked in a follow-up PR.

Reply:
Yep will do so now.

Reply:
https://github.com/lxc/lxd/pull/8836